client/src/no-match/index.scss

```diff
@@ -0,0 +1,5 @@
+.no-match {
+  p {
+    margin-top: 50px;
+  }
+}
```

Comment:
Not an important change. I just wanted to set us up for a place for Schalk to take over and do his magic.